src/OrbitGl/CaptureViewElement.h

```diff
@@ -9,6 +9,7 @@
 
 class TimeGraph;
 class GlCanvas;
+class Batcher;
```

Comment:
Will we need this in the future?

Reply:
It this actually needed right now? Why can't you include Batcher.h here?

Reply:
Not, isn't needed at all! Sorry, I probably saw the other classes and completed automatically without thinking.